packages/react-aria-components/docs/Modal.mdx

```diff
@@ -173,6 +174,12 @@ import {Modal, ModalOverlay} from 'react-aria-components';
 
 <ExampleList tag="modal" />
 
+## Starter kits
+
+To help kick-start your project, we offer starter kits that include example implementations of all React Aria components with various styling solutions. All components are fully styled, including support for dark mode, high contrast mode, and all UI states. Each starter comes with a pre-configured [Storybook](https://storybook.js.org/) that you can experiment with, or use as a starting point for your own component library.
+
+<StarterKits component="modal" tailwindComponent="alertdialog" />
```

Comment:
Forget if this was already brought up, but the Modal in the Vanilla CSS storybook doesn't cover the "Show Code" button as with the underlay. 

<img width="1152" alt="image" src="https://github.com/adobe/react-spectrum/assets/9661127/fb149766-489e-460b-8383-321e0ea1513b">

Reply:
pushed a fix to bump the z-index